src/app/services/storage.service.ts

```diff
@@ -105,8 +105,14 @@ export class StorageService {
 
   // Check to see if every VDEV has the same capacity. Best practices dictate every vdev should be uniform
   isMixedVdevCapacity(allVdevCapacities: Set<number>): boolean {
+    const max = Math.max(...allVdevCapacities);
+    const min = Math.min(...allVdevCapacities);
+    const diffPercentage = 100 - ((min / max) * 100);
     const diff = Math.max(...allVdevCapacities) - Math.min(...allVdevCapacities);
-    return allVdevCapacities.size > 1 && diff >= GiB * 2;
+
+    const isDiffNoticable = diffPercentage >= 5 && diff >= GiB * 2;
+
+    return allVdevCapacities.size > 1 && isDiffNoticable;
```

Comment:
Mostly out of curiosity: why do we need to check the `size > 1` here?  If as I understand it is a number of disks, then for one disk isDiffNoticable should always be false already.  If there are not disks, then I wonder what min/max functions return, or whether they throw an exception.

Reply:
Yea, that is what I meant. If there is only one value, it can't be much different from itself. ;)